store/mock.go

```diff
@@ -15,18 +15,18 @@ import (
 var _ Store = &MockStore{}
 
 type MockStore struct {
-	Blocks     map[uint32]block.Block
-	Accounts   map[crypto.Address]account.Account
-	Validators map[crypto.Address]validator.Validator
-	LastCert   *block.Certificate
-	LastHeight uint32
+	Blocks       map[uint32]block.Block
+	Accounts     map[crypto.Address]account.Account
+	ValidatorMap map[crypto.Address]validator.Validator
```

Comment:
I understand your point about renaming Validators to ValidatorMap. However, to maintain consistency with the rest of the code, we have two options:

1- Don't rename it.
2- Rename all variables.

I prefer the first option because it keeps the PR atomic. Using Validators here is shorter and more suitable for the `Mock` object.

Reply:
It is not allowed to declare Method `Validators()` because we have a `Validators` field in struct and the compiler does not allow that.

Reply:
Rename it to `ValidatorAddressess`